### PR TITLE
Estimated right margin

### PR DESCRIPTION
**Pull request by sujitacharya2005**
Resolves https://github.com/brave/brave-browser/issues/32736

## Submitter Checklist:

- [ ] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

Screenshot_20230905_173125
Screenshot_20230905_182735
Screenshot_20230905_173059

### CODE REVIEW

**Review by tapanmodh — APPROVED** [review]
lgtm

File: android/java/res/layout/wallet_balance_layout.xml [hunk 1]
```diff
@@ -64,7 +63,7 @@
         android:scaleType="fitXY"
         app:srcCompat="@drawable/tooltip_arrow"
         android:contentDescription="@null"
-        android:layout_marginEnd="16dp"
+        android:layout_marginEnd="46dp"
```
Comment (deeppandya) [inline android/java/res/layout/wallet_balance_layout.xml:66]: @sujitacharya2005 can you check on smaller screen, if the margin works there ?

File: android/java/res/layout/wallet_balance_layout.xml [hunk 2]
```diff
@@ -165,35 +165,35 @@
...
         app:layout_constraintBottom_toBottomOf="@id/ads_monthly_statement_text"
-        app:layout_constraintStart_toEndOf="@id/ads_monthly_statement_text"
+        app:layout_constraintEnd_toStartOf="@id/bat_ads_balance_learn_more_text"
         app:layout_constraintTop_toTopOf="@id/ads_monthly_statement_text" />
 
     <TextView
         android:id="@+id/bat_ads_balance_learn_more_text"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:layout_width="62dp"
```
Comment (deeppandya) [inline android/java/res/layout/wallet_balance_layout.xml:173]: we shouldn't set hard coded width here. it may cut the text for different translated text.

**Review by deeppandya — CHANGES_REQUESTED** [review]

File: android/java/res/layout/wallet_balance_layout.xml (hunk 2 above)
Comment (sujitacharya2005) [reply android/java/res/layout/wallet_balance_layout.xml]: ahh, It's my mistake.

File: android/java/res/layout/wallet_balance_layout.xml (hunk 1 above)
Comment (sujitacharya2005) [reply android/java/res/layout/wallet_balance_layout.xml:66]: This is working fine in my small screen device.
This is for triange position above estimated tooltip view.

**Review by deeppandya — APPROVED** [review]
LGTM

**Pull request merged**